examples/imdb/imdb/imdb/model_infer.py

```diff
@@ -0,0 +1,45 @@
+"""
+This model is adapted from https://www.tensorflow.org/tutorials/keras/text_classification
+The dataset used is IMDB taken from https://ai.stanford.edu/~amaas/data/sentiment/aclImdb_v1.tar.gz
+as seen in the tensorflow example
+"""
+import numpy as np
+import tensorflow as tf
+from tensorflow.keras import layers
+from tensorflow.keras.preprocessing.text import tokenizer_from_json
+import json
+from utils import standartize
+from tensorflow.keras.preprocessing.sequence import pad_sequences
+max_features = 10000
+sequence_length = 250
+AUTOTUNE = tf.data.AUTOTUNE
+embedding_dim = 16
+
+
+def tensorleap_model():
+    vectorized_inputs = tf.keras.Input(shape=250, dtype="int64")
+    model = tf.keras.Sequential([
+        layers.Embedding(max_features + 1, embedding_dim),
+        layers.Dropout(0.2),
+        layers.Dense(28, activation='relu'),
+        layers.GlobalAveragePooling1D(),
+        layers.Dropout(0.2),
+        layers.Dense(2, activation='sigmoid')])
+    x = vectorized_inputs
+    for lr in model.layers:
+        x = lr(x)
+    output = layers.Softmax(axis=-1)(x)
+    tl_model = tf.keras.Model(inputs=vectorized_inputs, outputs=output)
+    return tl_model
```

Comment:
lets move the creation of the model here to functional and fix the last layer to be only softmax

Reply:
This is done in [f98ef1f](https://github.com/tensorleap/tensorleap/pull/8/commits/f98ef1f091b6d63e3d85be0b974f9e9c7f176d9d)